skbuild/setuptools_wrap.py

```diff
@@ -358,6 +363,29 @@ def _load_cmake_spec():
         return None
 
 
+class SetupCfg:
+    """Parse `setup.cfg` (or equivalent) file to retrieve entries that matter to :mod:`skbuild`"""
+
+    relevant_options = (
+        "packages",
+        "package_data",
+        "package_dir",
+        "include_package_data",
+        "exclude_package_data",
+        "py_modules",
+        "data_files",
+        "entry_points",
+        "scripts",
+    )
+
+    def __init__(self, files: Optional[Iterable[str]] = None):
+        self.dist = Distribution()
+        self.dist.parse_config_files(files)
+
+    def to_dict(self) -> dict[str, Any]:
+        return {k: v for k in self.relevant_options if (v := getattr(self.dist, k))}
```

Comment:
```suggestion
        return {k:  getattr(self.dist, k) for k in self.relevant_options if getattr(self.dist, k, None)}
```

It does need to support Python 3.7 though. :D Also, this should not have worked, getattr without a default raises AttributeError, it does not return.

Reply:
I think this works because `Distribution` initializes the attributes to `None` (i've been using the fork for a little while). Anyway the edit is better.